azure-pipelines.yml

```diff
@@ -28,6 +28,15 @@ jobs:
       Python36Mac:
         imageName: 'macos-10.15'
         python.version: '3.6'
+      Python37Linux:
+        imageName: 'ubuntu-latest'
+        python.version: '3.7'
+      Python37Windows:
+        imageName: 'windows-latest'
+        python.version: '3.7'
+      Python37Mac:
+        imageName: 'macos-latest'
+        python.version: '3.7'
```

Comment:
It was just that 3.6, 3.8, 3.9 and 3.10 were all in there, but 3.7 wasn't, which looked like an accidental omission to me. Or is it deliberate and there is a good reason for it?

Reply:
I suspect it was left removed at some point because there weren't a lot of significant differences vs. 3.6 to save some time, but it's fine to add it back in. The CI isn't run as often for thinc and the overall test suite is a lot faster than spacy.

You could consider doing what we did for spacy, which is basically rotating through OSes for the earlier python versions with one OS per pre-3.10 python version. I would recommend making it a different rotation than with spacy to catch more problems with dependencies, which are mostly shared.

Reply:
I was about to recommend the "rotation" system as well :-)

Reply:
I also wondered why 3.6 is still in there — isn't it already past its EOL?

Reply:
I don't see any harm in keeping one 3.6 variant as long as it still runs on the CI

Reply:
Unresolving: I would go with Adriane's suggestion to do a rotation like in spaCy (but with permutated combinations) and have the 3.6 in here just once

Reply:
I've removed the change from this PR as Adriane has opened a separate PR for it.